pkg/ctl/utils/utils.go

```diff
@@ -15,8 +15,6 @@ func Command(flagGrouping *cmdutils.FlagGrouping) *cobra.Command {
 	cmdutils.AddResourceCmd(flagGrouping, verbCmd, updateClusterStackCmd)
 	cmdutils.AddResourceCmd(flagGrouping, verbCmd, updateKubeProxyCmd)
 	cmdutils.AddResourceCmd(flagGrouping, verbCmd, updateAWSNodeCmd)
-	cmdutils.AddResourceCmd(flagGrouping, verbCmd, updateCoreDNSCmd)
```

Comment:
`eksctl utils update-coredns` is to stay, only `eksctl utils install-coredns` needs to go, as that handles flipping from kube-dns to CoreDNS, which is needed during 1.10->1.11 upgrade.

Reply:
Reverted.